### DISCUSSION TITLE
请问如何自定义解析呢？

### DISCUSSION THREAD

**Discussion by echowings** [LEXUGE/dcompass#86] [Q&A]
比如
www.xxx.com. 123.123.123.123
或者
www.xxx.com 指定114.114.114.114来解析？

**Comment by LEXUGE** [comment]
可以直接用domain list来match并分流？

**Comment by LEXUGE** [comment]
最新的dcompass 版本支持编写和插入 A，AAAA，TXT以及EDNS record，可以参见README来编写。

**Comment by echowings** [comment]
很棒！
不过最新版有bug，Twitter的cdn地址会被解析到facebook，换回老版本就好了

**Reply by LEXUGE** [reply]
能file一个新的issue来报告一下这个bug吗，听起来挺严重的。
